inventory/group_vars/all.yml

```diff
@@ -7,6 +7,8 @@ bin_dir: /usr/local/bin
 # Where the binaries will be downloaded.
 # Note: ensure that you've enough disk space (about 1G)
 local_release_dir: "/tmp/releases"
+# Random shifts for retrying failed ops like pushing/downloading
+retry_stagger: 5
```

Comment:
It's not a stagger, but it's a splay. The distribution is between 0 and 5 and is random